INSTALL.linux.rst

```diff
@@ -0,0 +1,123 @@
+=====
+Linux
+=====
+
+
+  Tested on Arch Linux, Linux Mint 20, Ubuntu 20.04 and Fedora 33 (all amd64)
+
+
+Build Requirements
+------------------
+
+- cmake
```

Comment:
I think it'd be nice to separate prerequisites for libavif from prerequisites for this library -- specifically for prerequisites / install instructions for libavif I'd love to link to upstream instructions rather than rewriting them. Those are more likely to stay up to date as libavif itself evovles.